code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -327,6 +324,289 @@ datum/reagent/toxin/staminatoxin/on_mob_life(mob/living/carbon/M)
+	name = "Polonium"
+	id = "polonium"
+	description = "Cause significant Radiation damage over time."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.125 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+datum/reagent/toxin/polonium/on_mob_life(var/mob/living/M as mob)
+	M.radiation += 4
+	..()
+
+datum/reagent/toxin/histamine
+	name = "Histamine"
+	id = "histamine"
+	description = "A dose-dependent toxin, ranges from annoying to incredibly lethal."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	overdose_threshold = 30
+	toxpwr = 0
+
+datum/reagent/toxin/histamine/on_mob_life(var/mob/living/M as mob)
+	if(prob(50))
+		switch(pick(1, 2, 3, 4))
+			if(1)
+				M << "<span class='danger'>You can barely see!</span>"
+				M.eye_blurry = 3
+			if(2)
+				M.emote("cough")
+			if(3)
+				M.emote("sneeze")
+			if(4)
+				if(prob(75))
+					M << "You scratch at an itch."
+					M.adjustBruteLoss(2*REM)
+	..()
+
+datum/reagent/toxin/histamine/overdose_process(var/mob/living/M as mob)
+	M.adjustOxyLoss(1*REM)
+	M.adjustBruteLoss(1*REM)
+	M.adjustToxLoss(1*REM)
+	..()
+
+datum/reagent/toxin/formaldehyde
+	name = "Formaldehyde"
+	id = "formaldehyde"
+	description = "Deals a moderate amount of Toxin damage over time. 10% chance to decay into 10-15 histamine."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	toxpwr = 0.5
+
+datum/reagent/toxin/formaldehyde/on_mob_life(var/mob/living/M as mob)
+	if(prob(5))
+		M.reagents.add_reagent("histamine",pick(5,15))
+		M.reagents.remove_reagent("formaldehyde",1)
+	..()
+
+datum/reagent/toxin/venom
+	name = "Venom"
+	id = "venom"
+	description = "Will deal scaling amounts of Toxin and Brute damage over time. 15% chance to decay into 5-10 histamine."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+datum/reagent/toxin/venom/on_mob_life(var/mob/living/M as mob)
+	toxpwr = 0.05*volume
+	M.adjustBruteLoss((0.1*volume)*REM)
+	if(prob(15))
+		M.reagents.add_reagent("histamine",pick(5,10))
+		M.reagents.remove_reagent("venom",1)
+	..()
+
+datum/reagent/toxin/neurotoxin2
+	name = "Neurotoxin"
+	id = "neurotoxin2"
+	description = "Deals toxin and brain damage up to 60 before it slows down, causing confusion and a knockout after 54 elapsed cycles."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+datum/reagent/toxin/neurotoxin2/on_mob_life(var/mob/living/M as mob)
+	if(M.brainloss + M.toxloss <= 60)
+		M.adjustBrainLoss(1*REM)
+		M.adjustToxLoss(1*REM)
+	if(current_cycle == 54)
+		M.sleeping += 5
+	..()
+
+datum/reagent/toxin/cyanide
+	name = "Cyanide"
+	id = "cyanide"
+	description = "Deals toxin damage, alongside some oxygen loss. 8% chance of stun and some extra toxin damage."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.125 * REAGENTS_METABOLISM
+	toxpwr = 0.75
```

Comment:
I didn't change anything, cynaide/on_mob_life() already had adjustToxLoss(0.75), I simply moved it to be dealt in toxin/on_mob_life()